binaries/coordinator/src/lib.rs

```diff
@@ -40,6 +40,7 @@ mod tcp_utils;
 
 pub async fn start(
     port: Option<u16>,
+    external_events: impl Stream<Item = Event> + Unpin,
```

Comment:
Why do we need this?

Reply:
This is necessary for the `multiple-daemon` example where we use external events to check for connected machines. I did not want to revisit the logic there.
See: https://github.com/dora-rs/dora/blob/cee9294a209783f4ab7e0fe044a8e80a5a3e39fe/examples/multiple-daemons/run.rs#L35-L38